### DISCUSSION TITLE
Propuesta para ahorrar instanciaciones y otros problemas de encapsulamiento

### DISCUSSION THREAD

**Discussion by AstorArico72** [AstorArico72/Proyecto-Transversal-1#10] [Ideas]
Mi idea es ésta: Colocar todos los métodos para seleccionar e insertar los alumnos y materias en la clase Universidad que también contendrá el main.
Los métodos para conectar con la base SQL serán public static void.
Recomendaría usar `System.err.println ("Excepción encontrada: " + ex.getMessage ()) //Donde ex es un objeto de la clase java.lang.throwable ya instanciado en la sentencia catch de un bloque try-catch` para marcar los errores que estamos cometiendo.

**Comment by AstorArico72** [comment]
Ésto nos ahorraría problemas a la hora de diseñar la aplicación gráfica usando javax.swing.

**Comment by PabloBarrionuevo** [comment]
Esta bien, pero continuemos usando el modelo que ya tenemos:
Paquetes: 
- BD (todo lo necesario para conectarse y usar las clases data)
- Recursos (todo lo logico o clases que usaremos para contener la info obtenida del uso del anterior)
- TPTransversal (este si queres le podes cambiar el nombre, acá esta el main)
- Vistas (seria toda la interfaz de usuario visual UI)

El uso de métodos estáticos lo vemos mas adelante cuando hagamos la parte visual. Si fuera necesario.
